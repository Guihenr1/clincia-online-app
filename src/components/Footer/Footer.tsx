import { FC, ReactNode } from "react";
import useStyles from "./styles";
import { Box } from "@mui/material";

interface FooterProps {}

const Footer: FC<FooterProps> = () => {
  const s = useStyles();

  return <Box className={s.classes.footer}>@2023 Clinica Online</Box>;
};

Footer.defaultProps = {};

export default Footer;
