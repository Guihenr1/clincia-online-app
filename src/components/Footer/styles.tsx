import { makeStyles } from "tss-react/mui";

const useStyles = makeStyles()((theme) => ({
  footer: {
    fontSize: "13px",
    textAlign: "center",
    backgroundColor: theme.palette.primary.main,
    color: theme.palette.primary.contrastText,
    height: "10vh",
    display: "flex",
    alignItems: "center",
    justifyContent: "center",
  },
}));

export default useStyles;
