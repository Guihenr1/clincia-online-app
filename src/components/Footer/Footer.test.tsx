import { render, screen } from "@testing-library/react";
import Footer from "./Footer";
import "@testing-library/jest-dom/extend-expect";

describe("Footer", () => {
  test("renders the Footer", () => {
    const { container } = render(<Footer />);

    expect(screen.getByText("@2023 Clinica Online")).toBeInTheDocument();

    expect(container).toMatchSnapshot();
  });
});
