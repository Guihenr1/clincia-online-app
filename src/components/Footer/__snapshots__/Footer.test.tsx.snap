// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders the Footer 1`] = `
<div>
  <div
    class="MuiBox-root css-38qcwg-footer"
  >
    @2023 Clinica Online
  </div>
</div>
`;
